modules/mobile/docs/openapi.yaml

```diff
@@ -4432,7 +4511,7 @@ paths:
       summary: /v0/user/gender_identity
   /v0/user/gender_identity/edit:
     get:
-      description: Retrieves a list of valid gender identity keys. Note that this endpoint does not use the camel case key inflection header like most other mobile endpoints to keep the keys upcase. 
```

Comment:
git is giving me issues trying to just re-add this space so I'm just going to keep it.

Reply:
Git makes some good points.